### ISSUE TITLE
logs: lists of log entries should be a table

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#9310]
The logs page uses nested divs to recreate the look of a table. Instead, it should use table tags to perform the same feat.

The largest problem with using nested div soup is losing any semantic meaning. This means the mock table is a usability and accessibility nightmare. Using tables for tabular data fixes this.

Also, using tables could also improve rendering performance. In browsers, tables are heavily optimized, _especially_ when `table-layout: fixed;` is applied to the table via CSS. (Fixed layout causes sizing calculations to be performed on the first row and then applied to all the rest, instead of taking all rows into account and reflowing the table periodically.)

**Comment by garrett** [comment]
I tried adjusting the code to make it a table, and I couldn't get it working.

If it is indeed not so easy to switch, then we'll need to repurpose the divs to act more like a table using `role` and `aria-`* attributes.

Reference: https://www.w3.org/TR/wai-aria-practices-1.1/examples/table/table.html
